db/migrate/20210910101052_create_ecf_ineligible_participants.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class CreateECFIneligibleParticipants < ActiveRecord::Migration[6.1]
+  def change
+    create_table :ecf_ineligible_participants, id: :uuid do |t|
+      t.string :trn, index: { unique: true }
```

Comment:
What about when a single TRN has both reasons? Presumably it's fine only to use one?

Reply:
Good point probably easiest to have a reason that reflects both rather than 2 entries

Reply:
I've added a reason that reflects both and handle it so it sets both validation flags when evaluated in `ParticipantValidationService`